go.sum

```diff
@@ -16,6 +16,7 @@ cloud.google.com/go v0.56.0/go.mod h1:jr7tqZxxKOVYizybht9+26Z/gUq7tiRzu+ACVAMbKV
 cloud.google.com/go v0.57.0/go.mod h1:oXiQ6Rzq3RAkkY7N6t3TcE6jE+CIBBbA36lwQ1JyzZs=
```

Comment:
can we clean this up by regenerating?

Reply:
I think the current state is after the `go mod tidy`. Regenerating the mod again might create issues. WDYT?